### PR TITLE
Updating the what's new page

### PR DESCRIPTION
**Pull request by sfshaza2**
As it says

### CODE REVIEW

File: src/release/whats-new.md [hunk 1]
```diff
@@ -22,6 +25,50 @@ and review the [Dart changelog][].
...
+
+You might also check out the [Dart 3.3 release][] blog post.
+
+[3.19-tech]: {{site.flutter-medium}}/whats-new-in-flutter-3-19-58b1aae242d2
+[3.19-umbrella]: {{site.flutter-medium}}/starting-2024-strong-with-flutter-and-dart-cae9845264fe
+[Dart 3.3 release]: {{site.medium}}/dartlang/new-in-dart-3-3-extension-types-javascript-interop-and-more-325bf2bf6c13
+
+**Docs updated or added since the 3.16 release**
+
+* A new page on [migrating from Material 2 to Material 3][]
```
Comment (parlough) [inline src/release/whats-new.md:42]: This is missing a link definition. I guess you mean https://docs.flutter.dev/release/breaking-changes/material-3-migration?

File: src/release/whats-new.md [hunk 2]
```diff
@@ -22,6 +25,50 @@ and review the [Dart changelog][].
...
+  is added. Thanks to [@TahaTesser][] for writing this guide.
+* Material 3 uses theming in new and different ways than
+  Material 2. The [Use themes to share colors and font styles][]
+  cookbook recipe is updated to reflect these changes.
+* The [Flutter install][] pages have been updated. Please
+  [let us know][] if you have any feedback.
```
Comment (parlough) [inline src/release/whats-new.md:48]: "Let us know" is missing a link definition as well.

**Review by parlough — APPROVED** [review]
Thanks @sfshaza2. Two small link issues:

File: src/release/whats-new.md (hunk 1 above)
Comment (sfshaza2) [reply src/release/whats-new.md:42]: I fixed that a few minutes ago?

**Comment by sfshaza2** [comment]
Thaks, Parker! I believe I've fixed all missing links.

**Pull request merged**